contact.css

```diff
@@ -0,0 +1,110 @@
+@import "base.css";
+
+.contentDiv {
+  overflow-y: scroll;
+  width: 395px;
+  height: 660px;
+}
+.field-set {
```

Comment:
Try to be consistent with your naming conventions #camelCase 🐫

Reply:
Copy and Paste case!

.field-set (changed fieldSet)

Thx

Reply:
Update class deleted.

Forgot to delete related class in contact.css  ;)

But will focus on consistency ;)